src/heitech.zer0mqXt.core/Transport/Message.cs

```diff
@@ -11,25 +11,25 @@ namespace heitech.zer0mqXt.core.transport
     internal abstract class Message<TMessage>
         where TMessage : class
     {
-        protected ISerializerAdapter _serializer;
-        protected SocketConfiguration _configuration;
-        protected byte[] TypeFrame => _serializer.Serialize(typeof(TMessage).TypeFrameName());
+        protected readonly ISerializerAdapter Serializer;
+        protected SocketConfiguration Configuration;
```

Comment:
thought: In general, I prefer `protected` (readonly) properties over `protected` (readonly) fields.
But since the containing type is non-public, it's not that critical.